src/data/content/design-system/componenti/select.yaml

```diff
@@ -81,7 +84,18 @@ tabs:
             title: Come usarlo
             headingLevel: 2
             text: |
-              _In stesura_
+              - Imposta una opzione come predefinita la prima volta che l'utente incontra il componente, può essere uno dei valori o una prima opzione neutra che guidi la scelta dell'utente, ad esempio: "Scegli tra le opzioni: ". 
```

Comment:
```suggestion
              - Imposta un'opzione come predefinita la prima volta che l'utente incontra il componente: può essere uno dei valori o una prima opzione neutra che guidi la scelta dell'utente, ad esempio: "Scegli tra le opzioni: ". 
```

Reply:
In pratica sì. Solo che nel caso della Select di solito è una guida alla cosa da fare più che un segnaposto di esempio no?